docs/cloud/getting-started-with-airbyte-cloud.md

```diff
@@ -102,23 +96,24 @@ For more information, see [Connections and Sync Modes](../understanding-airbyte/
 
 To set up a connection:
 
-1. On the Airbyte Cloud dashboard, click **Connections** in the left navigation bar. 
-2. In the top right corner, click **+ new connection**. 
-3. On the New Connection page, select a source:
-    - To use an existing source, select your desired source from the Source dropdown. Click **Use existing source**. 
+1. On the Airbyte Cloud dashboard, click **Connections** and then click **+ New connection**.
+2. On the New connection page, select a source:
+    - To use an existing source, select your desired source from the **Source** dropdown. Click **Use existing source**. 
 
-    -  To set up a new source, enter a name for the new source and select the source from the Source type dropdown. The fields relevant to your source are displayed. Click **Setup Guide** for help with filling in the fields for your selected source. Click **Set up source**.
+    -  To set up a new source, select the source you want to set up from the **Source type** dropdown. The fields relevant to your source are displayed. The Setup Guide provides information to help you fill out the fields for your selected source. Click **Set up source**.
 
-4. Select a destination:
-    - To use an existing destination, select your desired destination from the Destination dropdown. Click **Use existing destination**. 
-    - To set up a new destination, enter a name for the new destination and select the destination from the Destination type dropdown. The fields relevant to your destination are displayed. Click **Setup Guide** for help with filling in the fields for your selected source. Click **Set up destination**.
+3. Select a destination:
+    - To use an existing destination, select your desired destination from the **Destination** dropdown. Click **Use existing destination**.
+    - To set up a new destination, select the source you want to set up from the **Destination type** dropdown. The fields relevant to your destination are displayed. The Setup Guide provides information to help you fill out the fields for your selected source. Click **Set up destination**.
```

Comment:
nit: replace source with destination in "select the source you want to set up" and "selected source"